### PR TITLE
GEODE-7309: Upgrade Lucene to 7.1.0

### PR DESCRIPTION
**Pull request by nabarunnag**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-core/src/main/java/org/apache/geode/internal/cache/InternalCacheForClientAccess.java [hunk 1]
```diff
@@ -1215,6 +1216,14 @@ public InternalQueryService getInternalQueryService() {
     return delegate.getInternalQueryService();
   }
 
+  @Override
+  public boolean hasMemberOlderThan(Version version) {
```
Comment (gesterzhou) [inline geode-core/src/main/java/org/apache/geode/internal/cache/InternalCacheForClientAccess.java]: You don't have to create a new method in GemFireCacheImpl. 

LuceneServiceImpl has the validateAllMembersAreTheSameVersion()

**Comment by nabarunnag** [comment]
@nabarunnag testing zapier

**Comment by nabarunnag** [comment]
@nabarunnag

**Comment by mkevo** [comment]
Hi @nabarunnag , what is the status of this activity? Do you need some help?

**Comment by onichols-pivotal** [comment]
this PR appears to be abandoned, can it be closed?

**Comment by nabarunnag** [comment]
this one has a unknow header serialization issue, it is also an issue with the 1.13 release blocker. Will revisit is ASAP once the related releases blockers are done.

**Comment by onichols-pivotal** [comment]
are release blockers done now so this can be revisited?

**Comment by onichols-pivotal** [comment]
This PR has been inactive for some while, can it be closed?

**Comment by onichols-pivotal** [comment]
This PR has been open for some while with no recent activity.  Please keep Geode tidy by closing PRs you're not actively using.

**Comment by onichols-pivotal** [comment]
closing as abandoned.  feel free to re-open or create a new PR from this branch if this picks up again.

**Pull request closed without merging**